docs/subscriptions/index.md

```diff
@@ -58,6 +58,14 @@ Datetime deleted
 }
 :::
 
+:::{admonition} Tombstoning
+:class: note
+
+As is implicit from the description of the servers-side behavior for the different calls, servers SHOULD hold all previous `guid` and `feed_url` field data with a link to the succeeding data (such that a path of values can be followed) or with a link to the most recent data. This in order to handle situations where clients submit old data. For example:
+* A user finds a podcast, whose URL had changed, and adds the old URL in the app. Because the client does not have that URL in its database, it recognizes the podcast as 'new' and POSTs this old (not-current) `feed_url` to the /subscriptions endpoind. If the user is already subscribed to the podcast (with the current feed URL) this would lead to a duplicate subscription.
```

Comment:
```suggestion
* A user finds a podcast, whose URL had changed, and adds the old URL in the app. Because the client doesn't have the old URL in its database, it recognizes the podcast as **new** and POSTs the `feed_url` to the `/subscriptions` endpoint. If the user is already subscribed to the podcast (with the current feed URL) this would lead to a duplicate subscription.
```